Cowles.js

```diff
@@ -0,0 +1,316 @@
+		item = new Zotero.Item("journalArticle");
+		item.title = doc.evaluate("//strong[contains(., 'CFP Paper Title')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		item.publicationTitle = doc.evaluate("//strong[contains(., 'Journal')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		item.date = doc.evaluate("//strong[contains(., 'CFP Date')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		item.volume = doc.evaluate("//strong[contains(., 'CFP Vol(Issue)')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent.split('(')[0];
+		item.issue = doc.evaluate("//strong[contains(., 'CFP Vol(Issue)')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent.split('(')[1][0];
+		item.pages = doc.evaluate("//strong[contains(., 'CFP page numbers')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		author = doc.querySelectorAll("div.comma span.comma a");
+		for (let auth of author) item.creators.push(ZU.cleanAuthor(auth.textContent,"author",false));
+		item.url = doc.evaluate("//strong[contains(., 'CFP Paper Title')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.getAttribute('href');
+		item.libraryCatalog = "Cowles Foundation";
+		try {
+			item.extra = "See also: " + doc.evaluate("//strong[contains(., 'See CFDP')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		}
+		catch (err) {}
+	}
+	else if (url.includes('/cfdp/')) {
+		item = new Zotero.Item("report");
+		item.title = doc.querySelector('h3 a').textContent;
+		item.reportType = "Cowles Foundation Discussion Paper";
+		item.reportNumber = doc.querySelector("#page-title").textContent.match(/\d+/)[0];
+		author = doc.querySelectorAll("div.comma span.comma a");
+		for (let auth of author) item.creators.push(ZU.cleanAuthor(auth.textContent,"author",false));
+
+		item.libraryCatalog = "Cowles Foundation";
+		item.date = doc.evaluate("//strong[contains(., 'Publication Date')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		item.pages = doc.evaluate("//strong[contains(., 'Pages')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().nextSibling.textContent;
+		try {
+			item.abstractNote = doc.evaluate("//strong[contains(., 'Abstract')]", doc, null, XPathResult.ANY_TYPE, null).iterateNext().parentElement.nextSibling.textContent;
+		}
+		catch (err) {}
+		pdfurl = doc.querySelector('h3 a').getAttribute('href');
+		item.attachments.push({
+			title: item.title,
+			mimeType:"application/pdf",
+			url:pdfurl
+		});
+	}
+	else if (url.includes('/cfm-')) {
+		item = new Zotero.Item("book");
+		item.title = doc.querySelector('a[href*="/pub/"]').textContent;
+		item.series = "Cowles Monograph";
+		item.seriesNumber = doc.querySelector("#page-title").textContent.match(/\d+/)[0];
+		item.libraryCatalog = "Cowles Foundation";
+		
+		var status = "";
```

Comment:
could we give this variable a better name? `creatorType` or something like that? And I'd just set it to "author" right here